content/20.results.md

```diff
@@ -24,7 +24,7 @@ Therefore, without first and middle names, we do not have author gender predicti
 We observed a slow increase of the proportion of predicted female authors, arriving at just over 20% in 2019 (Fig. {@fig:gender_breakdown}, left).
 We observe very similar trend within each journal, but estimated female proportion has increased the least in _PLOS Computational Biology_ (see [notebook](https://greenelab.github.io/iscb-diversity/09.visualize-gender.html#sup_fig_s1)).
 ISCB Fellows and keynote speakers appear to be more evenly split between men and women compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}, right); however, it has not yet reached parity.
-Further, taking all the years together, a Welch two-sample t-test does not reveal any statistically significant difference in the mean probability of ISCB speakers predicted to be female compared to that of authors ($t_{418} = 0.753$, $p = 0.226$).
+Further, taking all the years together, a Welch two-sample t-test did not reveal any statistically significant difference in the mean probability of ISCB speakers predicted to be female compared to that of authors (t~418~ = 0.753, p = 0.226).
```

Comment:
```suggestion
Further, taking all the years together, a Welch two-sample t-test did not reveal any statistically significant difference in the mean probability of ISCB speakers predicted to be female compared to that of authors (t~418~ = 0.753, _p_ = 0.226).
```

very nitpick https://stats.stackexchange.com/a/875/74908

Reply:
I was gonna let the journal applies their style but OK.